src/Creuna.Episerver.RedirectHandler.TestApp/modules/_protected/Creuna.Episerver.RedirectHandler/Core/Configuration/RedirectConfiguration.cs

```diff
@@ -47,7 +48,7 @@ public class RedirectConfiguration
         private bool? _fallbackToEPiServerError;
         private static int _loggingBufferSize = -1;
         private static int _loggingThresholdSize = -1;
-
+        private static int _redirectsLoggingAccumulationTime = -1;
```

Comment:
it is supposed to be seconds, I'll rename to  _redirectsLoggingAccumulationTimeSeconds